src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/domainevents/DomainEventDescriber.kt

```diff
@@ -42,10 +45,25 @@ class DomainEventDescriber(
       DomainEventType.APPROVED_PREMISES_MATCH_REQUEST_WITHDRAWN -> buildMatchRequestWithdrawnDescription(domainEventSummary)
       DomainEventType.APPROVED_PREMISES_REQUEST_FOR_PLACEMENT_CREATED -> buildRequestForPlacementCreatedDescription(domainEventSummary)
       DomainEventType.APPROVED_PREMISES_PLACEMENT_APPLICATION_ALLOCATED -> buildPlacementApplicationAllocatedDescription(domainEventSummary)
+      DomainEventType.APPROVED_PREMISES_ASSESSMENT_INFO_REQUEST -> buildInfoRequestDescription(domainEventSummary)
       else -> throw IllegalArgumentException("Cannot map ${domainEventSummary.type}, only CAS1 is currently supported")
     }
   }
 
+  private fun buildInfoRequestDescription(domainEventSummary: DomainEventSummary): String? {
+    val event = domainEventService.getFurtherInformationRequestMadeEvent(domainEventSummary.id())
+
+    return event.describe { data ->
+      val assessmentClarificationNote = assessmentClarificationNoteRepository.findByIdOrNull(data.eventDetails.requestId)
+        ?: throw RuntimeException("Clarification note with ID ${data.eventDetails.requestId} not found")
+
+      """
+        A further information request was made to the applicant:
+        "${assessmentClarificationNote.query}"
```

Comment:
so we're planning on keeping the details of the request for further info out of the domain event, but retrieving it when we build the timeline. This seems like a good solution!